README.md

```diff
@@ -109,11 +109,80 @@ This means that if a group of people all specify the same *moderation key*,
 they will collectively see the same set of administrators, moderators, and
 banned users.
 
+### cabal.ban(key, [opts], [cb])
+
+Bans a specific user, by their public key.
+
+Valid `opts` include:
+
+- `channel` (string, default="@") - the channel to ban the user from. `"@"` is cabal-wide.
+- `reason` (string, default="") - reason for the banishment.
+
+### cabal.unban(key, [opts], [cb])
+
+Unban a user by their public key.
+
+Valid `opts` include:
+
+- `channel` (string, default="@") - the channel to ban the user from. `"@"` is cabal-wide.
+- `reason` (string, default="") - reason for the banishment.
+
+### cabal.addMod(key, [opts], [cb])
+
+Add a moderator by their `key`.
+
+Moderators can ban and unban users who are not admins or moderators.
+
+Valid `opts` include:
+
+- `channel` (string, default="@") - the channel to ban the user from. `"@"` is cabal-wide.
```

Comment:
does the underlying view allow adding a mod/admin for a single channel?

nitpick: says ban. same case for the next four segments containing `channel`